### ISSUE TITLE
resource.landingPage.present

### ISSUE THREAD

**Issue by gothub** [NCEAS/metadig-checks#75]
## Description

Check that a dataset landing page is present

## Priority

- FAIR: Required

## Issues
- the landing page URL is checked for resolvability - should this be split into another check, i.e. one for concept presence, one for resolvability? 
   - if two checks are needed, should they both be in the Accessibility category?

## Procedure

- extract the dataset landing page from metadata and confirm that it is a URL

**Comment by gothub** [comment]
More details regarding populating EML documents with a landing page URL are discussed in [this issue](https://github.com/NCEAS/metadig-checks/issues/9).

**Comment by cscully-allison** [comment]
This should be changed to an accessibility issue, by consensus of workshop participants in the Interoperable group.

**Comment by gothub** [comment]
@cscully-allison thx, the FAIR category has been changed to "accessible".

**Comment by gothub** [comment]
Note that the resolvability part of this check is currently disabled in the FAIR suite, until  https://redmine.dataone.org/issues/881 is closed and the fix is incorporated into a DataONE CCI release.

**Comment by tedhabermann** [comment]
The target for this check exists in the DataCite Dialect: data[*].attributes.url

**Comment by gothub** [comment]
The current XPath for this check is
```
          /eml/dataset/distribution/online/url[@function="information"] |
          /eml/dataset/*/physical/distribution/online/url[@function="information"] 
          /*/distributionInfo/MD_Distribution/distributor/MD_Distributor/distributorTransferOptions/MD_DigitalTransferOptions/onLine/CI_OnlineResource[function/CI_OnLineFunctionCode/@codeListValue="information"]/linkage/URL/text()[normalize-space()] |
          /*/distributionInfo/MD_Distribution/transferOptions/MD_DigitalTransferOptions/onLine/CI_OnlineResource[function/CI_OnLineFunctionCode='information']/linkage/URL/text()[normalize-space()] |
          /*/identificationInfo/*/citation/CI_Citation/onlineResource/CI_OnlineResource[function/CI_OnLineFunctionCode='information']/linkage/URL/text()[normalize-space() ]
```

Remove this part of the XPath as it applies to a data entity and not the resource as a whole:
```
          /eml/dataset/*/physical/distribution/online/url[@function="information"] 
```

**Comment by gothub** [comment]
Removed entity XPath in commit 7f07670f91ac2322eaf36234d5a124cb90cba066
